test/integration/targets/prepare_nxos_tests/tasks/main.yml

```diff
@@ -1,6 +1,9 @@
+# NXAPI is enabled differently on NX-OS platforms.
```

Comment:
Is this an issue that could be fixed by a change to the `nxos_nxapi` module? I'm not really clear on what the issue is or why only providing `sandbox` sometimes helps